src/ticketing/use-get-fare-products-query.tsx

```diff
@@ -0,0 +1,13 @@
+import {getFareProducts} from '@atb/ticketing';
+import {useQuery} from '@tanstack/react-query';
+
+const ONE_HOUR_MS = 1000 * 60 * 60;
+
+export const useGetFareProductsQuery = () => {
+  return useQuery({
+    queryKey: ['getProducts'],
+    queryFn: getFareProducts,
```

Comment:
Could maybe use initData from `const {preassignedFareProducts} = useFirestoreConfiguration();`
See: https://tanstack.com/query/v4/docs/framework/react/guides/initial-query-data
Then no need for `useEffect` or `useState` in `TicketTabNav_PurchaseTabScreen`

Reply:
Updated, thank you for the suggestion!